package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "veritable-ui",
-  "version": "0.1.15",
+  "version": "0.2.0",
```

Comment:
A question on package version number, I can see that in each PR this number is incremented, but by how much? Is there any reading material on this?

Reply:
Standard SEMVER.

Reply:
It roughly follows [semver](https://semver.org/) but because we're still at version `0.x.x` we can do breaking changes whenever and it's all a bit meaningless. As we progress I'd expect a minor bump (like here) for major feature implementations and patch bumps for things that are more incremental.

This all becomes a bit more meaningfull once we hit v1 and at that moment we need to maintain the "interface" of the application